gregoriotex/details.html

```diff
@@ -158,9 +153,30 @@
 		<td><span class="gregoriocolor">gregoriocolor</span> (<span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span>)</td>
 	</tr>
 	<tr>
-	    <td></td>
-	    <td></td>
-	    <td>none (Plain <span class="tex">T<span class="epsilon">e</span>X</span>)</td>
+	    <td><code>annotation</code></td>
+	    <td>the annotation</td>
+	    <td>none</td>
+	</tr>
+	<tr>
+	    <td><code>commentary</code></td>
+	    <td>the commentary</td>
+	    <td><i>footnote-size italics</i></td>
```

Comment:
I haven't checked, but I'm pretty sure that there's a difference in the formatting for PlainTeX here as changing font size is very difficult in PlainTeX and so we don't generally do it.